api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1414,10 +1414,12 @@ type ContainerOverrides struct {
 
 	// PeerVerificationRules provides the rules for what client certificates
 	// the process should accept.
+	// +kubebuilder:validation:MaxLength=3000
```

Comment:
We should ensure that this length is good enough for our use cases.

Reply:
I checked this on one of our QA cluster. The value was something around 2600. So I put 3k here.